beetsplug/lastgenre/genres-tree.yaml

```diff
@@ -156,6 +156,31 @@
     - timba
     - twoubadou
     - zouk
+- classical:
+    - ballet
+    - baroque:
+        - baroque music
+    - cantata
+    - chamber music:
+        - string quartet
+    - classic
```

Comment:
No, `classic rock` is listed elsewhere in the genre tree, but the tag is likely to be used for all sorts of things on Last.fm, not just classical music.